client/state/users/suggestions/reducer.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	USER_SUGGESTIONS_RECEIVE,
+	USER_SUGGESTIONS_REQUEST,
+	USER_SUGGESTIONS_REQUEST_SUCCESS,
+	USER_SUGGESTIONS_REQUEST_FAILURE,
+} from 'state/action-types';
+
+/**
+ * Returns the updated requests state after an action has been dispatched. The
+ * state maps site ID keys to whether a request for user suggestions is in progress.
+ *
+ * @param  {Object} state   Current state
+ * @param  {Object} action  Action payload
+ * @return {Object}         Updated state
+ */
+export function requesting( state = {}, action ) {
+	switch ( action.type ) {
+		case USER_SUGGESTIONS_REQUEST:
+		case USER_SUGGESTIONS_REQUEST_SUCCESS:
+		case USER_SUGGESTIONS_REQUEST_FAILURE:
+			return {
+				...state,
+				[ action.siteId ]: USER_SUGGESTIONS_REQUEST === action.type,
+			};
+	}
+
+	return state;
+}
+
+/**
+ * Returns the updated items state after an action has been dispatched. The
+ * state maps site ID keys to an array of user suggestions.
+ *
+ * @param  {Object} state   Current state
+ * @param  {Object} action  Action payload
+ * @return {Object}         Updated state
+ */
+export function items( state = {}, action ) {
```

Comment:
As is, we'll be persisting both fetching state and suggestions in browser storage since we aren't handling calypso's special `SERIALIZE` and `DESERIALIZE` actions. If you don't wish to persist anything, we can return default state ( in this case {} ) for both action types. See also:
https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#data-persistence--2754-

We do have a few utils that reduce some boilerplate.

So a few options here. We can use [createReducer](https://github.com/Automattic/wp-calypso/blob/master/client/state/utils.js#L167) that has some default persistence handlers:
```
const initialState = {};
const customHandlers = {
    [ USER_SUGGESTIONS_RECEIVE ] : ( state, action ) => { 
        return {
            ...state,
            [ action.siteId ]: action.suggestions,
        };
    }
};
const schema = null; // if you don't want to persist anything. Otherwise create a json schema. (Ping me for q's or help)
createReducer( initialState, customHandlers, schema );
```
Or we can add these cases to the switch:
```
    case SERIALIZE:
        return {};
    case DESERIALIZE:
        return {};
```